target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -115,6 +115,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan:1" "1:lan:2" "2:lan:3" "3:lan:4" "6@eth0"
 		;;
+	mikrotik,rb760igs)
```

Comment:
mt7621 has been moved to kernel 5.4, which changed switch setup entirely. Just have a look at what's done for rb750gr3 now.